beta/src/content/community/team.md

```diff
@@ -18,10 +18,22 @@ Current members of the React team are listed in alphabetical order below.
     Andrew got started with web development by making sites with WordPress, and eventually tricked himself into doing JavaScript. His favorite pastime is karaoke. Andrew is either a Disney villain or a Disney princess, depending on the day.
 </TeamMember>
 
+<TeamMember name="Andrey Lunyov" permalink="andrey-lunyov" photo="/images/team/andrey-lunyov.jpg" github="alunyov" twitter="alunyov" title="Engineer at Meta">
+    I started my career as a designer and then gradually transitioned into web development. After joining the React Data team at Meta, I worked on adding an incremental JavaScript compiler to Relay, and then later on, I worked on removing the same compiler from Relay. Outside of work, I like to play music and engage in various sports.
```

Comment:
@alunyov Should we do 3rd person to be consistent?

Reply:
yes we should

Reply:
edited